BrainCloudClient/tests/TestFile.cs

```diff
@@ -46,7 +46,10 @@ public void TestSimpleUpload()
             _bc.Client.RegisterFileUploadCallbacks(FileCallbackSuccess, FileCallbackFail);
 
             FileInfo info = new FileInfo(CreateFile(4024));
-
+            Stream buffInfo = new FileStream(info.FullName, FileMode.Open);
+            byte[] data = ConvertToBytes(buffInfo);
+            _bc.Client.FileService.FileStorage.Add(info.Name, data);
```

Comment:
Hum, no. Customer don't add stuff to your internal cache. He just calls "UploadFile" or "UploadFileFromMemory". We handling the cache internally